categories.py

```diff
@@ -0,0 +1,68 @@
+categories = [{"keyword": "project manager", "title": "Project Management"},
```

Comment:
Potentially think about changing the structure to be more like this
```json
{
    "title": "Engineering"
    "keywords": [
        "mechanical engineer",
        "electrical engineer",
        "technologist",
        "fpga"
    ]
}
```
What you have now should still work fine, I just think this would make it a bit easier to manage if anything needs to be changed in the future.